hscontrol/handlers.go

```diff
@@ -71,37 +71,34 @@ func (h *Headscale) VerifyHandler(
 		Str("handler", "/verify").
 		Msg("verify client")
 
-	body, err := io.ReadAll(req.Body)
-	if err != nil {
-		log.Error().
-			Str("handler", "/verify").
-			Err(err).
-			Msg("Cannot read request body")
-		http.Error(writer, "Internal error", http.StatusInternalServerError)
-		return
-	}
+	doVerify := func() (bool, error) {
```

Comment:
Can you make it a func so it doesnt have to be instantiated every time the verify handler is called?

Reply:
I don't think it is to be instantiated every time, this function should be inline, which is the most basic optimization. It's hard to believe Golang wouldn't do it.

But if you request to do so, then I will fix it.